framework/structures.md

```diff
@@ -467,7 +468,7 @@ the beacon key can be discarded.
 
 ##### Encryption Context
 
-The [custom encryption context](#encryption-context) associated with this beacon key.
+The [encryption context](#encryption-context) associated with this beacon key.
```

Comment:
Why is the encryption context of the branch key prepended but the beacon key is not?